Sources/main.cpp

```diff
@@ -19,6 +22,33 @@
 #include "core/FastIsType.h"
 #include "AutogenMetadata.h"
 
+QFile logs("error.log");
```

Comment:
Logfile should be unique for each game session and have the start datetime in the name. You can look in the `Chat.cpp` for hints - `QUuid` will be helpful here. Overall, name should be something like `log-<datetime>-<uuid>.txt`